src/mondeath.cpp

```diff
@@ -654,6 +653,18 @@ void mdeath::broken( monster &z )
 
     g->m.add_item_or_charges( z.pos(), broken_mon );
 
+
+    // adds ammo drop
+    const bool drops_ammo = z.type->has_flag( MF_DROPS_AMMO );
```

Comment:
```suggestion
    if( z.type->has_flag( MF_DROPS_AMMO ) ) {
```
No need for a separate variable.